### DISCUSSION TITLE
Can't update, failed config, file corrupt

### DISCUSSION THREAD

**Discussion by Fierofiend** [syssi/esphome-jk-bms#255] [Q&A] [answered]
I've been using this for months now with no issue.  However after updating to the latest ESP home, I tried to update thru HA settings rather than the ESP-home interface and I received this error.  I don't know if its related to the issue.  

![image](https://user-images.githubusercontent.com/24301394/220149161-a49562f4-b55d-41d9-9162-2c2795443707.png)

Any suggestions?

**Comment by syssi** [comment, marked as answer]
Please try a clean build. It's a local issue on your side for sure. Please double check you harddrive/sdd/sd-card. May be your hardware is dying.

**Reply by Fierofiend** [reply]
I went ahead and just deleted all the BMS related files in HA, and reloaded my yaml from scratch.  It compiled/installed no problem.  Thanks!
